TAA/TAA.md

```diff
@@ -0,0 +1,141 @@
+* a. With regard to all Transactions, Transaction Author will:
+    1) comply with any requirements imposed by the Transaction Endorser on the Transaction Author and any Transactions endorsed by the Transaction Endorser;
+    2) not write Transactions containing Personal Data until Sovrin Foundation approves Public Write Access and permits Transactions to contain Personal Data pursuant to Section 3(b) below.
+* b. If the Sovrin Foundation approves Public Write Access and permits Transaction Authors to write Transactions that contain Permissible Personal Data ("**Personal Data Transactions**"), then Transaction Author expressly agrees that:
+    1) It will not write any Transactions that contain Impermissible Personal Data to the Sovrin Ledger Networks;
+    2) it is an independent data controller (as defined in the GDPR) of the Personal Data Transactions and will be responsible for the lawfulness of the Processing of such data in compliance with the Data Protection Laws;
+    3) it acknowledges and will notify all data subjects whose Personal Data it Processes that functions inherent in blockchain technology may render fulfilling data subject requests difficult or impossible. For example, due to blockchain’s immutability, data stored on a blockchain generally cannot be removed or altered once the data is confirmed on the blockchain;
+    4) it irrevocably waives any and all claims, rights and/or obligations it may have now or in the future against the Sovrin Foundation as a result of being unable to fulfill data subject requests in accordance with Data Protection Laws;
+    5) it agrees to be bound by the terms and conditions applicable to Transaction Author as a controller of Personal Data Transactions under the Data Processing Agreements with Stewards and Transaction Endorsers, each in their roles as processors under the GDPR, and attached as Appendices A and B (the "**DPAs**");
+    6) that the Sovrin Foundation has the right to enter into the DPAs on its behalf and the DPAs are made a part of the Agreement in their entirety;
+    7) by signing this Agreement, each Party is deemed to have signed the DPAs, including the Standard Contractual Clauses with the Sovrin Foundation and Transaction Author as the "Data exporter", and with either a Steward or a Transaction Endorser as "Data importer", as applicable;
+    8) at the Sovrin Foundation’s request, the Transaction Author will reimburse the Sovrin Foundation for any costs incurred by the Sovrin Foundation in enforcing the Transaction Author’s rights under the GDPR, including but not limited to fulfillment of data subject rights, rights of oversight and audit, etc.; and
+    9) it irrevocably waives any and all claims that it may have now or in the future that the Sovrin Foundation lacks the rights to enter into the DPAs on its behalf and bind Transaction Author to the DPAs’ terms and conditions, including the limitation of liability therein.
+
+## **4)Term and Termination**
```

Comment:
A space between the `)` and the following text would be good for each of these headers. Also, I don't think making these headers bold is visually any different in most markdown rendering engines and I'd recommend removing it for readability.

Reply:
agreed.